src/CwpSearchResult.php

```diff
@@ -62,15 +61,15 @@ class CwpSearchResult extends ViewableData
     public function __construct($terms = '', ArrayData $results = null)
     {
         $this->query = $terms;
-        $searchableService = SearchableService::singleton();
```

Comment:
I'm presuming this API got introduced in FullText 3.7?

Reply:
That's correct